braindecode/models/conformer.py

```diff
@@ -0,0 +1,279 @@
+# Authors: Yonghao Song <eeyhsong@gmail.com>
+#
+# License: BSD (3-clause)
+
+import math
+
+import torch
+import torch.nn.functional as F
+from einops import rearrange
+from einops.layers.torch import Rearrange, Reduce
+from torch import nn, Tensor
+
+
+class EEGConformer(nn.Sequential):
+    """EEG Conformer.
+
+    Convolutional Transformer for EEG decoding.
+
+    The paper and original code with more details about the methodological
+    choices are availible at the [EEG Conformer]_ and [EEG Conformer Code]_.
+
+    This neural network architecture recieves a different input shape than
+    other models. The input shape should be four-dimensional matrix
+    representing the EEG signals.
+
+         (batch_size, 1, n_channels, n_timesteps)`.
```

Comment:
This is not true, as line 156 unsqueezes the 3D input to 4D.
```suggestion
         (batch_size, n_channels, n_timesteps)`.
```